concise_concepts/conceptualizer/Conceptualizer.py

```diff
@@ -347,7 +317,9 @@ def add_patterns(input_dict):
                     words = [
                         "".join(
                             [
-                                token.lemma_ if token.lemma_ else token.text
+                                token.lemma_ + token.whitespace_
+                                if token.lemma_
+                                else token.text
```

Comment:
This ensures that if any tokens have whitespace, the whitespace is preserved. Very useful attribute (https://spacy.io/api/token#attributes)